paparazzi/src/main/java/app/cash/paparazzi/accessibility/AccessibilityRenderExtension.kt

```diff
@@ -169,6 +169,7 @@ public class AccessibilityRenderExtension : RenderExtension {
     val mainAccessibilityText =
       config.getOrNull(SemanticsProperties.ContentDescription)?.joinToString(", ")
         ?: config.getOrNull(SemanticsProperties.Text)?.joinToString(", ")
+        ?: config.getOrNull(SemanticsProperties.EditableText)?.text
```

Comment:
does `joinToString` not make sense for this case?

Reply:
It's just a string, as opposed `List<String>` returned by content description and text.